starter-page-templates.php

```diff
@@ -44,5 +44,16 @@ function page_templates_enqueue() {
 	}
 
 	wp_enqueue_script( 'starter-page-templates' );
+	
+	$default_info = array(
+		'title' => get_bloginfo( 'name' ),
+	);
+	$site_info = get_site_option( 'site_contact_info', array() );
+	
+	$config = array(
+		'siteInformation' => array_merge( $default_info, $site_info ),
+		'vertical' => get_site_option( 'site_vertical' ),
```

Comment:
Let's fix #24 first. The misaligned arrows should have triggered a phpcs error :)